benchmark/README.md

```diff
@@ -0,0 +1,60 @@
+# Duck-Read-Cache-FS Performance Benchmark Suite
+
+## Overview
+This benchmark suite evaluates the performance of our DuckDB fs extension compared to the standard DuckDB httpfs. 
+Key aspects measured include:
+
+* Read performance from cloud storage (S3)
+* Caching efficiency for frequently accessed data (working)
+* Query execution time with cached vs non-cached data (working)
+* Memory usage and disk utilization (working)
+* Compatibility with existing DuckDB httpfs workflows 
+
+> **Note**
+> Our goal is to provide a more efficient, caching-aware alternative to DuckDB's httpfs while maintaining full compatibility.
```

Comment:
What do you mean by cache-aware?
https://en.wikipedia.org/wiki/Cache-oblivious_algorithm